misk-admin/src/main/kotlin/misk/web/metadata/config/ConfigMetadata.kt

```diff
@@ -0,0 +1,49 @@
+package misk.web.metadata.config
+
+import com.google.inject.Provider
+import jakarta.inject.Inject
+import misk.config.AppName
+import misk.config.MiskConfig
+import misk.resources.ResourceLoader
+import misk.web.metadata.Metadata
+import misk.web.metadata.jvm.JvmMetadataAction
+import wisp.deployment.Deployment
+
+data class ConfigMetadata(
+  val resources: Map<String, String?>
+) : Metadata(id = "config", metadata = resources)
+
+class ConfigMetadataProvider : Provider<ConfigMetadata> {
+  @Inject @AppName private lateinit var appName: String
+  @Inject private lateinit var deployment: Deployment
+  @Inject private lateinit var config: wisp.config.Config
+  @Inject private lateinit var jvmMetadataAction: JvmMetadataAction
+  @Inject private lateinit var mode: ConfigMetadataAction.ConfigTabMode
+
+  override fun get() = ConfigMetadata(
+    resources = generateConfigResources(appName, deployment, config)
+  )
+
+  private fun generateConfigResources(
```

Comment:
Honest question – is there a good way to get the actually-parsed config from the application bring-up, rather than re-implementing it? I don’t like gathering the metadata again here.